src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -127,13 +127,15 @@ def test_set_read_dev_agreement_error(self):
             assert doc(selector).text() == 'This field is required.'
 
     def test_read_dev_agreement_skip(self):
+        set_config('last_dev_agreement_change_date', '2019-06-10 00:00')
         after_agreement_last_changed = (
             datetime(2019, 6, 10) + timedelta(days=1))
         self.user.update(read_dev_agreement=after_agreement_last_changed)
         response = self.client.get(reverse('devhub.submit.agreement'))
         self.assert3xx(response, reverse('devhub.submit.distribution'))
 
     def test_read_dev_agreement_set_to_future(self):
```

Comment:
Can you suffix that method with `_fallback` so that it's more obvious that it's what it's testing ?

Reply:
It's not testing the fallback. It's testing that the agreement page is skipped because the developer agreed to it one day later than what is set in the config.

Reply:
Yes, but *that* is using the fallback, since the date set in the config is in the future.

Reply:
53adef4cda